<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormularForm</class>
 <widget class="QWidget" name="FormularForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>510</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="boSave">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>440</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Записать</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="bnCancel">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>440</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>185</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>ЗАМЕТКИ</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1016</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color:darkgray;</string>
   </property>
   <property name="text">
    <string>ФОРМУЛЯР</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="eNotes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>998</width>
     <height>191</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="undoRedoEnabled">
    <bool>true</bool>
   </property>
   <property name="lineWrapMode">
    <enum>QPlainTextEdit::NoWrap</enum>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextEditable</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>1001</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,10,0,0">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ОПЕРАТОР</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="3">
     <widget class="QLineEdit" name="eOperator"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>ЛОКОМОТИВ</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QLineEdit" name="eLocomotive"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>ДЕПО</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="3">
     <widget class="QLineEdit" name="eDepot"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>ЛЕВОЕ КОЛЕСО: №</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="eLeftWheelN"/>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>ДИАМЕТР</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLineEdit" name="eLeftWheelD">
      <property name="inputMethodHints">
       <set>Qt::ImhFormattedNumbersOnly</set>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>ПРАВОЕ КОЛЕСО: №</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="eRightWheelN">
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>ДИАМЕТР</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="eRightWheelD">
      <property name="inputMethodHints">
       <set>Qt::ImhFormattedNumbersOnly</set>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
